lms/templates/courseware/courseware-chromeless.html

```diff
@@ -0,0 +1,159 @@
+<%! from django.utils.translation import ugettext as _ %>
```

Comment:
I really dislike the idea of adding in a new base template if we can avoid it, because it means that we now have to keep two templates in sync with changes, and everyone has to understand and know how to test this. Is there a way to not do this?